src/modules/Dropdown/Dropdown.js

```diff
@@ -46,20 +46,11 @@ export default class Dropdown extends Component {
       PropTypes.string,
     ]),
 
-    /** Array of `{ text: '', value: '' }` options */
+    /** Array of Dropdown.Item props e.g. `{ text: '', value: '' }` */
     options: customPropTypes.every([
       customPropTypes.disallow(['children']),
       customPropTypes.demand(['selection']),
-      PropTypes.arrayOf(PropTypes.shape({
-        value: PropTypes.oneOfType([
-          PropTypes.string,
-          PropTypes.number,
-        ]).isRequired,
```

Comment:
Note: This `isRequired` got dropped in this change.

On the one hand, `PropTypes.arrayOf(PropTypes.shape(DropdownItem.propTypes))` is much DRYer and less brittle.

On the other, there's a slight difference in terms of what props are required that we're now ignoring.

Open to suggestions here.